Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel_calibration.xml

```diff
@@ -1,39 +1,29 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
-       xmlns:xs="http://www.w3.org/2001/XMLSchema"
-       xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+<?xml version="1.0" ?><lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0" xmlns:xs="http://www.w3.org/2001/XMLSchema" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
 
-  <info name="FCCee_ECalBarrel_calibration"
-        title="Settings for Calibration of Inclined ECal Barrel Calorimeter"
-        author="M.Aleksa,J.Faltova,A.Zaborowska, V. Volkl"
-        url="no"
-        status="development"
-        version="1.0">
+  <info author="M.Aleksa,J.Faltova,A.Zaborowska, V. Volkl" name="FCCee_ECalBarrel" status="development" title="Settings for FCCee Inclined ECal Barrel Calorimeter" url="no" version="1.0">
     <comment>
-      Calibration hits setup for the inclined EM calorimeter.
+      Settings for the inclined EM calorimeter.
```

Comment:
Hi Jana, I totally agree! Actually this is the reason why this PR modifies the calibration and upstream xml's. Here is the script that takes the original xml and write the other two automatically: [link](https://github.com/BrieucF/LAr_scripts/blob/main/FCCSW_ecal/write_calibration_xml.py)) (I did not push it here because it is probably not the right place and because it does also other things relying on my folder organization).

Reply:
OK, perfect! It will be nice if a similar script appears in the FCCSW somewhere. Could be done in the next PR.